test/server/cards/03_TWI/units/IndependentSenator.spec.ts

```diff
@@ -0,0 +1,39 @@
+describe('Independent Senator', function () {
+    integration(function (contextRef) {
+        describe('Independent Senator\'s ability', function () {
+            it('should heal 2 damage from a unit or a base', function () {
```

Comment:
Wrong test name. Also, if there's just one case you can ditch the `describe` and just have the entire name in the `it()`